modules/mri_violations/php/mri_violations.class.inc

```diff
@@ -213,7 +213,24 @@ class Mri_Violations extends \NDB_Menu_Filter_Form
         //      then uses it as is, the path is correct.
         //   3. if it is not in the assembly dir, then it is in the trashbin,
         //      so we fix the path with that in mind.
-        $this->query    = " FROM (
+
+        // Find which sites the user has access to
+        $accessibleSites = \User::singleton()->hasPermission('access_all_profiles')
```

Comment:
This doesn't seem like the right permission.. isn't there an imaging related all profiles permission?

Reply:
@cmadjar maybe you know?

Reply:
There are two permissions for imaging browser: imaging_browser_view_allsites and imaging_browser_phantom_allsites.

Not sure which one is best or if both should be used...

Reply:
So what should we do? Just merge it with access_all_profiles?

Reply:
maybe we could use `dicom_archive_view_allsites`? Just realized that this is the one used by the DICOM archive module. @driusan what do you think? 

Makes more sense than access_all_profiles and less complex than the imaging browser permissions

Reply:
I don't have any strong opinion. We could also create a new permission.